jdaviz/core/template_mixin.py

```diff
@@ -473,10 +472,6 @@ def _on_viewer_changed(self, msg=None):
             self._on_layers_changed()
 
     def _valid_layer(self, viewer, layer):
```

Comment:
Is there a plan to use this for future work or can this be removed?

Reply:
that's a good point, thanks for catching that.  I originally left it in thinking it was used more widely and needed to be overridden, but that isn't the case it seems - so I'll remove it and can always re-implement it if we need it.